docs/en/helm/known-issues.md

```diff
@@ -0,0 +1,111 @@
+Helm is highly recommended that new charts are created via `helm create` command
+as the template names are automatically defined as per this best practice.  You
+must let your main workload use the templated full name as its name.
+DO NOT assign any value to `.Values.fullnameOverride`.
+
+## Upgrade an application
+
+By contrast to CUE based workload, application using Helm schematic workload
+cannot benefit from [application level rollout](https://github.com/oam-dev/kubevela/blob/master/design/vela-core/rollout-design.md#applicationdeployment-workflow) because they use totally different mechanisms to create and 
+manage workloads.
+So currently [application inplace upgrade](https://github.com/oam-dev/kubevela/blob/master/design/vela-core/rollout-design.md#application-inplace-upgrade-workflow) is the only one choice for users who
+want to upgrade their applications containing Helm schematic workloads.
+Just as [the sample](./trait.md#update-an-applicatiion) shows, users can modify
+the application's configuration directly to upgrade it.
+
+#### Changing settings will trigger Helm release upgrade
+
+For Helm schematic workload, `.spec.components.settings` in the config of
+application will override the default values of a Chart.
+Any changes applied to `settings` will trigger a Helm release upgrade.
+Most of this procedure is handled by Helm and Flux2/helm-controller,
+while KubeVela has no opinion on it, even upgrade has failed or timeout.
+
+On one hand, users should be very clear with what will happen after making such
+an upgrade to the Chart release. On the other hand, users can define remediation
+strategies in the Helm schematic according to [fluxcd/helmrelease API
+doc](https://github.com/fluxcd/helm-controller/blob/main/docs/api/helmrelease.md#upgraderemediation)
+and [spec doc](https://toolkit.fluxcd.io/components/helm/helmreleases/#configuring-failure-remediation) 
+in case of upgrade failure.
+
+For example
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: webapp-chart
+spec:
+...
+  schematic:
+    helm:
+      release:
+        chart:
+          spec:
+            chart: "podinfo"
+            version: "5.1.4"
+        upgrade:
+          remediation:
+            retries: 3 
+            remediationStrategy: rollback
+      repository:
+        url: "http://oam.dev/catalog/"
+
+```
+
+Currently, users can hardly get helpful information of a living Helm release to
+figure out what happened if upgrading failed.  
+We will enhance the observability to help users track the situation of Helm
+release in application level.
+
+#### Changing traits may make Pod restart
+
+Traits work on Helm schematic workload in the same way as CUE based workload.
+The changes on traits will effect the main workload finally.
+Users should pay attention that, pods belonging to the workload may restart
+because of various possible changes applied to the workload, that will result 
+in service down.
```

Comment:
```suggestion
Users should pay attention that, pods belonging to the workload may restart twice on upgrade,
one is by the helm upgrade, and the other one is because of various possible changes applied to the workload,
that will result in service down.
```